assets/js/base/components/cart-checkout/shipping-rates-control/index.js

```diff
@@ -84,6 +88,22 @@ const ShippingRatesControl = ( {
 		}
 	}, [ shippingRatesLoading, shippingRates ] );
 
+	// Prepare props to pass to the ExperimentalOrderShippingPackages slot fill.
+	// We need to pluck out receiveCart.
+	// eslint-disable-next-line no-unused-vars
+	const { extensions, receiveCart, ...cart } = useStoreCart();
+	const slotFillProps = {
+		className,
+		collapsible,
+		noResultsMessage,
+		renderOption,
+		extensions,
+		cart,
+		components: {
+			ShippingRatesControlPackage,
+		},
```

Comment:
Since this doesn't vary from the current implementation much (we just switch from import to prop) I don't want to tackle this here, but I am cc'ing @senadir as there may be other reasons why it was done this way initially.

Reply:
Yup I'm fine with that.